### DISCUSSION TITLE
The software crashed while zooming in on the Earth.

### DISCUSSION THREAD

**Discussion by luohuiiqng** [pelicanmapping/osgearth#2530] [Support Q&A]
Hello, the program crashed when I was constantly zooming the earth with the mouse (at the beginning of the program). What are the possible causes of this crash? The image map I used was a tile map posted online.
Thank you!
![微信图片_20240511154246](https://github.com/gwaldron/osgearth/assets/82392471/9019dcd3-120b-4972-8ddc-d85bfbf06fa7)

**Comment by gwaldron** [comment]
Unfortunately, there's not enough information here to reproduce the issue. If you can provide steps to reproduce that will help.

**Reply by luohuiiqng** [reply]
Thank you. This question is incidental. I'll let you know as soon as I find a pattern.
